tests/conftest.py

```diff
@@ -349,6 +330,21 @@ def client():
 
             db.session.commit()
 
+            generate_project_key_pair(users[2], units[0].projects[0])
+            generate_project_key_pair(users[2], units[0].projects[2])
```

Comment:
This could be refactored but no reason for you to do it.

Reply:
Yes that's a piece of what I mentioned in the daily. It's there for now and better not in the long run.